packages/canon/src/utils/getBreakpoint.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useMediaQuery } from '../hooks/useMediaQuery';
+import type { Breakpoint } from '../types';
+
+export const breakpoints: { name: string; id: Breakpoint; value: number }[] = [
+  { name: 'Initial', id: 'initial', value: 0 },
+  { name: 'Extra Small', id: 'xs', value: 640 },
+  { name: 'Small', id: 'sm', value: 768 },
+  { name: 'Medium', id: 'md', value: 1024 },
+  { name: 'Large', id: 'lg', value: 1280 },
+  { name: 'Extra Large', id: 'xl', value: 1536 },
+];
+
+export const getBreakpoint = (): Breakpoint => {
```

Comment:
Needs to be called `useBreakpoint` and be placed in `hooks` since it uses a hook itself.

Reply:
Done